tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -317,7 +317,16 @@ public function filterResponseByNotificationSubjectAndResource(string $subject,
 	 * @return void
 	 */
 	public function userShouldGetANotificationWithMessage(string $user, string $subject, TableNode $table):void {
-		$this->userListAllNotifications($user);
+		$count = 0;
+		// sometimes the test might try to get notification before the notification is created by the server
+		// in order to prevent test from failing because of that list the notifications again
+		while (!isset($this->filterResponseAccordingToNotificationSubject($subject)->message) && $count <= 5) {
+			\sleep(1);
```

Comment:
maybe sleep after the first request. If notifications are returned on the first try then we don't have to wait
```suggestion
			if ($count > 0) \sleep(1);
```